### PR TITLE
Add HSA2 content, points, 11/23 points

### PR DESCRIPTION
**Pull request by chrisalleng**

### CODE REVIEW

File: data/pilots/first-order/tie-se-bomber.json [hunk 1]
```diff
@@ -219,6 +219,37 @@
       "image": "https://infinitearenas.com/xw2/images/pilots/sienarjaemustestpilot.png",
       "keywords": ["TIE"],
       "epic": true
+    },
+    {
+      "name": "Jul Jerjerrod",
```
Comment (doweaver) [inline data/pilots/first-order/tie-se-bomber.json:224]: Need charges on Jul, right? Or maybe they're here and I'm just not seeing them?

File: data/pilots/separatist-alliance/hyena-class-droid-bomber.json [hunk 2]
```diff
@@ -192,6 +192,76 @@
...
+      "limited": 1,
+      "cost": 3,
+      "xws": "dbs404-siegeofcoruscant",
+      "ability": "While you perform an attack at range 1, you must roll 1 additional die. After the attack hits, suffer 1 [Critical Hit] damage.",
+      "shipAbility": {
+        "name": "Networked Calculations",
+        "text": "While you defend or perform an attack, you may spend 1 calculate token from a friendly ship at range 0-1 to change 1 [Focus] result to an [Evade] or [Hit] result."
+      },
+      "image": "https://infinitearenas.com/xw2/images/quickbuilds/dbs404-siegeofcoruscant.png",
+      "standardLoadout": ["plasmatorpedoes", "contingencyprotocol-siegeofcoruscant", "strutlockoverride-siegeofcoruscant"],
```
Comment (doweaver) [inline data/pilots/separatist-alliance/hyena-class-droid-bomber.json:239]:
```suggestion
      "standardLoadout": ["advprotontorpedoes", "contingencyprotocol-siegeofcoruscant", "strutlockoverride-siegeofcoruscant"],
```

File: data/pilots/separatist-alliance/hyena-class-droid-bomber.json [hunk 3]
```diff
@@ -192,6 +192,76 @@
       "artwork": "https://squadbuilder.fantasyflightgames.com/card_art/f58f50898f4fa3900eb1b7d01aec4ae5.jpg",
       "keywords": ["Droid"],
       "epic": true
+    },
+    {
+      "name": "DBS-32C",
+      "caption": "Siege of Coruscant",
+      "initiative": 3,
+      "limited": 1,
+      "cost": 3,
```
Comment (doweaver) [inline data/pilots/separatist-alliance/hyena-class-droid-bomber.json:201]:
```suggestion
      "cost": 3,
      "charges": { "value": 2, "recovers": 0 },
```

File: data/pilots/separatist-alliance/rogue-class-starfighter.json [hunk 4]
```diff
@@ -305,6 +305,35 @@
...
       "extended": true,
       "epic": true
+    },
+    {
+      "name": "Durge",
+      "caption": "On His Own Time",
+      "initiative": 5,
+      "limited": 1,
+      "cost": 4,
+      "loadout": 10,
```
Comment (doweaver) [inline data/pilots/separatist-alliance/rogue-class-starfighter.json:315]:
```suggestion
      "loadout": 10,
      "charges": { "value": 1, "recovers": 0 },
```

File: data/pilots/separatist-alliance/vulture-class-droid-fighter.json [hunk 5]
```diff
@@ -168,6 +168,116 @@
       "artwork": "https://squadbuilder.fantasyflightgames.com/card_art/5fb72145f183eeddd8d37ce1a4d114f7.jpg",
       "keywords": ["Droid"],
       "epic": true
+    },
+    {
+      "name": "DFS-081",
+      "caption": "Siege of Coruscant",
+      "initiative": 3,
+      "limited": 1,
+      "cost": 2,
```
Comment (doweaver) [inline data/pilots/separatist-alliance/vulture-class-droid-fighter.json:177]:
```suggestion
      "cost": 2,
      "charges": { "value": 2, "recovers": 0 },
```

File: data/pilots/first-order/tie-se-bomber.json (hunk 1 above)
Comment (chrisalleng) [reply data/pilots/first-order/tie-se-bomber.json:224]: fixed!

File: data/pilots/separatist-alliance/vulture-class-droid-fighter.json [hunk 6]
```diff
@@ -168,6 +168,116 @@
...
+        "name": "Networked Calculations",
+        "text": "While you defend or perform an attack, you may spend 1 calculate token from a friendly ship at range 0-1 to change 1 [Focus] result to an [Evade] or [Hit] result."
+      },
+      "ability": "After a friendly ship at range 0-1 skips its execute maneuver step, you may spend 1 [Charge]. If you do, if there is an asteroid or debris cloud at range 0 of it, that ship may repair 1 damage.",
+      "standard": true,
+      "extended": true,
+      "image": "https://infinitearenas.com/xw2/images/pilots/theironassembler.png",
+      "artwork": "https://infinitearenas.com/xw2/images/artwork/pilots/theironassembler.png",
+      "cost": 2,
+      "loadout": 5,
```
Comment (doweaver) [inline data/pilots/separatist-alliance/vulture-class-droid-fighter.json:248]:
```suggestion
      "loadout": 5,
      "charges": { "value": 3, "recovers": 0 },
```

File: data/upgrades/configuration.json [hunk 7]
```diff
@@ -751,5 +751,22 @@
...
+    "limited": 0,
+    "xws": "strutlockoverride-siegeofcoruscant",
+    "sides": [
+      {
+        "title": "strutlockoverride-siegeofcoruscant",
+        "type": "Configuration",
+        "ability": "At the start of your activation, you may spend 1 [Charge]. If you do, ignore obstacles while you move through them this round.",
+        "slots": ["Configuration"]
+      }
+    ],
```
Comment (doweaver) [inline data/upgrades/configuration.json:766]: Do the charges go on this...? Probably?
```suggestion
    ],
    "charges": { "value": 2, "recovers": 0 },
```

**Pull request merged**